### PR TITLE
Update ListenBrainz logos 

### PR DESCRIPTION
**Pull request by shivam-kapila**
As it says on the tin :)

### CODE REVIEW

**Review by yvanzo — APPROVED** [review]

File: metabrainz/static/img/projects/listenbrainz.svg [hunk 1]
```diff
@@ -1,202 +1 @@
...
...g><g><path d="M66.082 47.24c-10.12 0-20.365 3.656-26.736 9.543-6.886 6.361-10.521 16.245-10.348 27.989a3.99 3.99 0 00-3.41 3.941v18.119a4 4 0 004 4h2.033c.316 0 .619-.047.914-.115v1.602c0 2.209 1.565 4 3.495 4h1.898c1.93 0 3.495-1.791 3.495-4V82.65c0-2.209-1.565-4-3.495-4H36.03c-1.717 0-3.138 1.42-3.432 3.288.065-1.855.204-3.652.435-5.096 1.154-7.232 4.119-13.268 8.688-17.488 5.664-5.232 15.226-8.613 24.361-8.613 1.042 0 3.241.043 4.277.143v-3.49c-1.2-.117-3.163-.154-4.277-.154zm-34.459 59.188a2.652 2.652 0 01-2.651-2.65v-12.01a2.651 2.651 0 012.651-2.652h.913v17.312h-.913z" opacity=".25"/><path fill="#FFFEDB" d="M66.082 44.24c-10.12 0-20.365 3.656-26.736 9.543-6.886 6.361-10.521 16.245-10.348 27.989a3.99 3.99 0 00-3.41 3.941v18.119a4 4 0 004 4h2.033c.316 0 .619-.047.914-.115v1.602c0 2.209 1.565 4 3.495 4h1.898c1.93 0 3.495-1.791 3.495-4V79.65c0-2.209-1.565-4-3.495-4H36.03c-1.717 0-3.138 1.42-3.432 3.287.065-1.854.204-3.651.435-5.096 1.154-7.232 4.119-13.268 8.688-17.488 5.664-5.232 15.226-8.613 24.361-8.613 1.042 0 3.241.043 4.277.143v-3.49c-1.2-.116-3.163-.153-4.277-.153zm-34.459 59.188a2.652 2.652 0 01-2.651-2.65v-12.01a2.651 2.651 0 012.651-2.652h.913v17.312h-.913z"/></g></svg>
```
Comment (MonkeyDo) [inline metabrainz/static/img/projects/listenbrainz.svg:1]: This svg hasn't been generated at the same size as the other projects, and it appears a bit off and misaligned compared to the other icons on the projects page.
To fix this, you can add `width="154" height="154"` attributes and change the viewbox attribute to  `viewBox="-5 0 154 154"`

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="154" height="154" viewBox="-5 0 154 154"><path fill="#EB743B" d="M75.354 7.823v144l61-35v-74z"/><path fill="#353070" d="M70.354 7.823l-61 35v74l61 35z"/><g opacity=".07"><defs><path id="a" d="M75.354 33.239h51.433v90.792H75.354z"/></defs><clipPath id="b"><use xlink:href="#a" overflow="visible"/></clipPath><path clip-path="url(#b)" d="M92.657 99.557a2.1 2.1 0 011.819-1.03c.378 0 .752.104 1.078.297a2.12 2.12 0 01.735 2.898 2.097 2.097 0 01-1.817 1.032c-.379 0-.755-.105-1.084-.301a2.092 2.092 0 01-.964-1.294 2.1 2.1 0 01.233-1.602m12.602 22.362a2.121 2.121 0 01-2.981-.225 2.116 2.116 0 01.231-2.981 2.08 2.08 0 011.373-.512 2.117 2.117 0 012.109 2.278 2.086 2.086 0 01-.732 1.44M95.734 63.08a2.128 2.128 0 012.916-.656 2.12 2.12 0 01.66 2.919 2.115 2.115 0 01-2.919.661 2.128 2.128 0 01-.657-2.924m6.379-20.187c-.6.951-1.937 1.271-2.912.653l-.096-.062a2.107 2.107 0 01-.557-2.851 2.093 2.093 0 011.784-.98 2.11 2.11 0 011.781 3.24M118.88 54.491a2.096 2.096 0 011.783-.98c.4 0 .79.113 1.125.325.481.304.813.775.937 1.33a2.08 2.08 0 01-.28 1.584c-.597.951-1.938 1.273-2.912.653l-.093-.062a2.105 2.105 0 01-.56-2.85m0 23.463a2.096 2.096 0 011.783-.98c.4 0 .79.112 1.125.326.481.304.813.774.937 1.329a2.083 2.083 0 01-.28 1.585c-.597.95-1.938 1.271-2.912.652l-.093-.062a2.105 2.105 0 01-.56-2.85m0 26.303a2.099 2.099 0 011.783-.98c.4 0 .79.113 1.125.326.481.304.813.775.937 1.33a2.08 2.08 0 01-.28 1.584c-.597.951-1.938 1.271-2.912.653l-.093-.062a2.107 2.107 0 01-.56-2.851"/><path clip-path="url(#b)" d="M108.53 113.927a6.11 6.11 0 00-8.631-.662 6.12 6.12 0 00-2.087 5.396c-1.949.701-4.258 1.366-5.328 1.332-.822-.043-1.302-.345-2.224-.971-1.172-.795-2.418-1.512-4.149-1.847 2.936-2.761 6.009-6.83 8.584-12.834a6.15 6.15 0 005.039-2.972c1.724-2.896.773-6.659-2.124-8.394a6.133 6.133 0 00-3.134-.864 6.154 6.154 0 00-5.261 2.986 6.079 6.079 0 00-.678 4.636 6.075 6.075 0 002.302 3.429c-4.053 9.218-9.219 13.08-12.565 14.685a2.016 2.016 0 00-.361.168c-1.186.531-2.05.704-2.558.808v4c.275-.03 1.884-.118 4.247-1.192 5.347-1.357 6.839-.354 8.407.709 1.021.692 2.291 1.554 4.263 1.655.099.006.204.009.308.009 2.11 0 5.172-1.022 7.021-1.723a6.132 6.132 0 004.285 1.75 6.112 6.112 0 003.983-1.476 6.072 6.072 0 002.119-4.176 6.072 6.072 0 00-1.458-4.452M92.657 97.151a2.114 2.114 0 013.632 2.165 2.097 2.097 0 01-1.817 1.032c-.379 0-.755-.105-1.084-.301a2.092 2.092 0 01-.964-1.294 2.101 2.101 0 01.233-1.602m12.602 22.363a2.121 2.121 0 01-2.981-.225 2.116 2.116 0 01.231-2.981 2.08 2.08 0 011.373-.512 2.117 2.117 0 012.109 2.278 2.086 2.086 0 01-.732 1.44M106.304 38.027a6.088 6.088 0 00-2.708-3.844 6.12 6.12 0 00-3.264-.944 6.076 6.076 0 00-5.174 2.85 6.108 6.108 0 00.144 6.758c-3.646 4.124-8.707 4.314-9.943 4.293-3.692-1.604-7.234-2.164-10.005-2.317v3.996c2.442.144 5.553.686 8.748 2.136.005.002.01.002.015.005 3.445 1.562 6.334 3.927 8.607 7.041a6.364 6.364 0 00-.381.538c-1.796 2.854-.943 6.643 1.903 8.444.98.622 2.111.951 3.269.951a6.105 6.105 0 005.183-2.851c1.806-2.851.951-6.642-1.908-8.453-1.384-.872-3.092-1.114-4.667-.755-1.538-2.155-3.323-4.017-5.345-5.567 2.517-.778 5.401-2.268 7.78-5.095a6.18 6.18 0 001.77.266 6.08 6.08 0 005.172-2.848 6.053 6.053 0 00.804-4.604m-10.57 22.647a2.128 2.128 0 012.916-.656 2.12 2.12 0 01.66 2.919 2.115 2.115 0 01-2.919.661 2.128 2.128 0 01-.657-2.924m6.379-20.187c-.6.951-1.937 1.271-2.912.653l-.096-.062a2.107 2.107 0 01-.557-2.851 2.093 2.093 0 011.784-.98 2.11 2.11 0 011.781 3.24"/><path clip-path="url(#b)" d="M115 79.011a6.036 6.036 0 002.11 2.649 6.098 6.098 0 003.55 1.14 6.083 6.083 0 005.173-2.848 6.064 6.064 0 00.805-4.605 6.103 6.103 0 00-2.709-3.844 6.128 6.128 0 00-3.266-.944c-.125 0-.25.008-.375.018-.71-3.158-.919-7.769.035-11.265.113.011.225.024.337.024a6.08 6.08 0 005.173-2.848 6.062 6.062 0 00.805-4.604 6.103 6.103 0 00-2.709-3.844 6.128 6.128 0 00-3.266-.944 6.079 6.079 0 00-5.173 2.85 6.104 6.104 0 001.025 7.764c.021.02.048.037.071.059-1.349 4.518-1.145 10.324-.036 14.417-.144.132-.294.254-.426.399a6.327 6.327 0 00-.634.825 6.2 6.2 0 00-.71 1.578c-3.371.488-5.198 1.671-6.824 2.729-2.035 1.324-3.961 2.572-9.636 2.666-1.963-.23-3.552-.178-5.29-.116-1.43.052-2.91.105-4.848.003-1.335-.071-2.104-.882-3.458-2.424-1.803-2.055-4.219-4.648-9.369-5.022v4c3.295.299 4.796 1.887 6.356 3.667 1.487 1.694 3.175 3.615 6.256 3.782 2.118.114 3.76.052 5.208.001 2.491-.092 4.453-.161 7.855.683 2.608.644 12.099 9.674 14.774 14.323-.109.142-.219.282-.314.434-1.743 2.755-1.026 6.385 1.62 8.251.072.052.151.104.283.191.981.621 2.11.948 3.267.948a6.081 6.081 0 005.173-2.849 6.06 6.06 0 00.805-4.603 6.1 6.1 0 00-2.709-3.845 6.128 6.128 0 00-4.809-.749c-2.56-4.188-8.652-10.455-13.189-13.792 1.813-.628 3.05-1.436 4.212-2.188 1.427-.93 2.614-1.69 4.857-2.067m3.88-26.926a2.096 2.096 0 011.783-.98c.4 0 .79.113 1.125.325.481.304.813.775.937 1.33a2.08 2.08 0 01-.28 1.584c-.597.951-1.938 1.273-2.912.653l-.093-.062a2.105 2.105 0 01-.56-2.85m0 23.463a2.096 2.096 0 011.783-.98c.4 0 .79.112 1.125.326.481.304.813.774.937 1.329a2.083 2.083 0 01-.28 1.585c-.597.95-1.938 1.272-2.912.652l-.093-.062a2.105 2.105 0 01-.56-2.85m0 26.304a2.099 2.099 0 011.783-.98c.4 0 .79.113 1.125.326.481.304.813.775.937 1.33a2.08 2.08 0 01-.28 1.584c-.597.951-1.938 1.271-2.912.653l-.093-.062a2.107 2.107 0 01-.56-2.851"/></g><path fill="#D3562C" d="M108.53 116.927a6.11 6.11 0 00-8.631-.662 6.12 6.12 0 00-2.087 5.396c-1.949.701-4.258 1.366-5.328 1.332-.822-.043-1.302-.345-2.224-.971-1.172-.795-2.418-1.512-4.149-1.847 2.936-2.761 6.009-6.83 8.584-12.834a6.15 6.15 0 005.039-2.972c1.724-2.896.773-6.659-2.124-8.394a6.133 6.133 0 00-3.134-.864 6.154 6.154 0 00-5.261 2.986 6.079 6.079 0 00-.678 4.636 6.075 6.075 0 002.302 3.429c-4.053 9.218-9.219 13.08-12.565 14.685a2.016 2.016 0 00-.361.168c-1.186.531-2.05.704-2.558.808v4c.275-.03 1.884-.118 4.247-1.192 5.347-1.357 6.839-.354 8.407.709 1.021.692 2.291 1.554 4.263 1.655.099.006.204.009.308.009 2.11 0 5.172-1.022 7.021-1.723a6.132 6.132 0 004.285 1.75 6.112 6.112 0 003.983-1.476 6.072 6.072 0 002.119-4.176 6.072 6.072 0 00-1.458-4.452m-15.873-16.776a2.114 2.114 0 013.632 2.165 2.097 2.097 0 01-1.817 1.032c-.379 0-.755-.105-1.084-.301a2.092 2.092 0 01-.964-1.294 2.101 2.101 0 01.233-1.602m12.602 22.363a2.121 2.121 0 01-2.981-.225 2.116 2.116 0 01.231-2.981 2.08 2.08 0 011.373-.512 2.117 2.117 0 012.109 2.278 2.086 2.086 0 01-.732 1.44"/><path d="M84.355 53.434"/><path fill="#D3562C" d="M106.304 40.031a6.088 6.088 0 00-2.708-3.844 6.12 6.12 0 00-3.264-.944 6.076 6.076 0 00-5.174 2.85 6.108 6.108 0 00.144 6.758c-3.646 4.124-8.707 4.314-9.943 4.293-3.692-1.604-7.234-2.164-10.005-2.317v3.996c2.442.144 5.553.686 8.748 2.136.005.002.01.002.015.005 3.445 1.562 6.334 3.927 8.607 7.041a6.206 6.206 0 00-.381.538c-1.796 2.854-.943 6.643 1.903 8.444.98.622 2.111.951 3.269.951a6.105 6.105 0 005.183-2.851c1.806-2.851.951-6.642-1.908-8.453-1.384-.872-3.092-1.114-4.667-.755-1.538-2.155-3.323-4.017-5.345-5.567 2.517-.778 5.401-2.268 7.78-5.095a6.18 6.18 0 001.77.266 6.08 6.08 0 005.172-2.849 6.05 6.05 0 00.804-4.603m-10.57 22.647a2.128 2.128 0 012.916-.656 2.12 2.12 0 01.66 2.919 2.115 2.115 0 01-2.919.661 2.128 2.128 0 01-.657-2.924m6.379-20.187c-.6.951-1.937 1.271-2.912.653l-.096-.062a2.107 2.107 0 01-.557-2.851 2.093 2.093 0 011.784-.98 2.11 2.11 0 011.781 3.24"/><path fill="#D3562C" d="M115 82.011a6.036 6.036 0 002.11 2.649 6.098 6.098 0 003.55 1.14 6.081 6.081 0 005.173-2.849 6.063 6.063 0 00.805-4.604 6.103 6.103 0 00-2.709-3.844 6.128 6.128 0 00-3.266-.944c-.125 0-.25.008-.375.018-.71-3.158-.919-7.769.035-11.265.113.011.225.024.337.024a6.08 6.08 0 005.173-2.848 6.062 6.062 0 00.805-4.604 6.103 6.103 0 00-2.709-3.844 6.128 6.128 0 00-3.266-.944 6.079 6.079 0 00-5.173 2.85 6.104 6.104 0 001.025 7.764c.021.02.048.037.071.059-1.349 4.518-1.145 10.324-.036 14.417-.144.132-.294.254-.426.399a6.327 6.327 0 00-.634.825 6.2 6.2 0 00-.71 1.578c-3.371.488-5.198 1.671-6.824 2.729-2.035 1.323-3.961 2.571-9.636 2.665-1.963-.23-3.552-.178-5.29-.116-1.43.052-2.91.105-4.848.003-1.335-.071-2.104-.882-3.458-2.424-1.803-2.055-4.219-4.648-9.369-5.022v4c3.295.298 4.796 1.887 6.356 3.667 1.487 1.694 3.175 3.615 6.256 3.782 2.118.114 3.76.052 5.208.001 2.491-.092 4.453-.161 7.855.683 2.608.644 12.099 9.674 14.774 14.323-.109.142-.219.282-.314.434-1.743 2.755-1.026 6.385 1.62 8.251.072.052.151.104.283.191.981.621 2.11.948 3.267.948a6.081 6.081 0 005.173-2.849 6.06 6.06 0 00.805-4.603 6.1 6.1 0 00-2.709-3.845 6.128 6.128 0 00-4.809-.749c-2.56-4.188-8.652-10.455-13.189-13.792 1.813-.628 3.05-1.436 4.212-2.188 1.427-.929 2.614-1.689 4.857-2.066m3.88-26.926a2.096 2.096 0 011.783-.98c.4 0 .79.113 1.125.325.481.304.813.775.937 1.33a2.08 2.08 0 01-.28 1.584c-.597.951-1.938 1.273-2.912.653l-.093-.062a2.105 2.105 0 01-.56-2.85m0 23.463a2.096 2.096 0 011.783-.98c.4 0 .79.112 1.125.326.481.304.813.774.937 1.329a2.084 2.084 0 01-.28 1.585c-.597.95-1.938 1.271-2.912.652l-.093-.062a2.105 2.105 0 01-.56-2.85m0 26.304a2.099 2.099 0 011.783-.98c.4 0 .79.113 1.125.326.481.304.813.775.937 1.33a2.08 2.08 0 01-.28 1.584c-.597.951-1.938 1.271-2.912.653l-.093-.062a2.107 2.107 0 01-.56-2.851"/><path fill="#FFFEDB" d="M108.53 113.927a6.11 6.11 0 00-8.631-.662 6.12 6.12 0 00-2.087 5.396c-1.949.701-4.258 1.366-5.328 1.332-.822-.043-1.302-.345-2.224-.971-1.172-.795-2.418-1.512-4.149-1.847 2.936-2.761 6.009-6.83 8.584-12.834a6.15 6.15 0 005.039-2.972c1.724-2.896.773-6.659-2.124-8.394a6.133 6.133 0 00-3.134-.864 6.154 6.154 0 00-5.261 2.986 6.079 6.079 0 00-.678 4.636 6.075 6.075 0 002.302 3.429c-4.053 9.218-9.219 13.08-12.565 14.685a2.016 2.016 0 00-.361.168c-1.186.531-2.05.704-2.558.808v4c.275-.03 1.884-.118 4.247-1.192 5.347-1.357 6.839-.354 8.407.709 1.021.692 2.291 1.554 4.263 1.655.099.006.204.009.308.009 2.11 0 5.172-1.022 7.021-1.723a6.132 6.132 0 004.285 1.75 6.112 6.112 0 003.983-1.476 6.072 6.072 0 002.119-4.176 6.072 6.072 0 00-1.458-4.452M92.657 97.151a2.114 2.114 0 013.632 2.165 2.097 2.097 0 01-1.817 1.032c-.379 0-.755-.105-1.084-.301a2.092 2.092 0 01-.964-1.294 2.101 2.101 0 01.233-1.602m12.602 22.363a2.121 2.121 0 01-2.981-.225 2.116 2.116 0 01.231-2.981 2.08 2.08 0 011.373-.512 2.117 2.117 0 012.109 2.278 2.086 2.086 0 01-.732 1.44M106.304 38.027a6.088 6.088 0 00-2.708-3.844 6.12 6.12 0 00-3.264-.944 6.076 6.076 0 00-5.174 2.85 6.108 6.108 0 00.144 6.758c-3.646 4.124-8.707 4.314-9.943 4.293-3.692-1.604-7.234-2.164-10.005-2.317v3.996c2.442.144 5.553.686 8.748 2.136.005.002.01.002.015.005 3.445 1.562 6.334 3.927 8.607 7.041a6.364 6.364 0 00-.381.538c-1.796 2.854-.943 6.643 1.903 8.444.98.622 2.111.951 3.269.951a6.105 6.105 0 005.183-2.851c1.806-2.851.951-6.642-1.908-8.453-1.384-.872-3.092-1.114-4.667-.755-1.538-2.155-3.323-4.017-5.345-5.567 2.517-.778 5.401-2.268 7.78-5.095a6.18 6.18 0 001.77.266 6.08 6.08 0 005.172-2.848 6.053 6.053 0 00.804-4.604m-10.57 22.647a2.128 2.128 0 012.916-.656 2.12 2.12 0 01.66 2.919 2.115 2.115 0 01-2.919.661 2.128 2.128 0 01-.657-2.924m6.379-20.187c-.6.951-1.937 1.271-2.912.653l-.096-.062a2.107 2.107 0 01-.557-2.851 2.093 2.093 0 011.784-.98 2.11 2.11 0 011.781 3.24"/><path fill="#FFFEDB" d="M115 79.011a6.036 6.036 0 002.11 2.649 6.098 6.098 0 003.55 1.14 6.083 6.083 0 005.173-2.848 6.064 6.064 0 00.805-4.605 6.103 6.103 0 00-2.709-3.844 6.128 6.128 0 00-3.266-.944c-.125 0-.25.008-.375.018-.71-3.158-.919-7.769.035-11.265.113.011.225.024.337.024a6.08 6.08 0 005.173-2.848 6.062 6.062 0 00.805-4.604 6.103 6.103 0 00-2.709-3.844 6.128 6.128 0 00-3.266-.944 6.079 6.079 0 00-5.173 2.85 6.104 6.104 0 001.025 7.764c.021.02.048.037.071.059-1.349 4.518-1.145 10.324-.036 14.417-.144.132-.294.254-.426.399a6.327 6.327 0 00-.634.825 6.2 6.2 0 00-.71 1.578c-3.371.488-5.198 1.671-6.824 2.729-2.035 1.324-3.961 2.572-9.636 2.666-1.963-.23-3.552-.178-5.29-.116-1.43.052-2.91.105-4.848.003-1.335-.071-2.104-.882-3.458-2.424-1.803-2.055-4.219-4.648-9.369-5.022v4c3.295.299 4.796 1.887 6.356 3.667 1.487 1.694 3.175 3.615 6.256 3.782 2.118.114 3.76.052 5.208.001 2.491-.092 4.453-.161 7.855.683 2.608.644 12.099 9.674 14.774 14.323-.109.142-.219.282-.314.434-1.743 2.755-1.026 6.385 1.62 8.251.072.052.151.104.283.191.981.621 2.11.948 3.267.948a6.081 6.081 0 005.173-2.849 6.06 6.06 0 00.805-4.603 6.1 6.1 0 00-2.709-3.845 6.128 6.128 0 00-4.809-.749c-2.56-4.188-8.652-10.455-13.189-13.792 1.813-.628 3.05-1.436 4.212-2.188 1.427-.93 2.614-1.69 4.857-2.067m3.88-26.926a2.096 2.096 0 011.783-.98c.4 0 .79.113 1.125.325.481.304.813.775.937 1.33a2.08 2.08 0 01-.28 1.584c-.597.951-1.938 1.273-2.912.653l-.093-.062a2.105 2.105 0 01-.56-2.85m0 23.463a2.096 2.096 0 011.783-.98c.4 0 .79.112 1.125.326.481.304.813.774.937 1.329a2.083 2.083 0 01-.28 1.585c-.597.95-1.938 1.272-2.912.652l-.093-.062a2.105 2.105 0 01-.56-2.85m0 26.304a2.099 2.099 0 011.783-.98c.4 0 .79.113 1.125.326.481.304.813.775.937 1.33a2.08 2.08 0 01-.28 1.584c-.597.951-1.938 1.271-2.912.653l-.093-.062a2.107 2.107 0 01-.56-2.851"/><g opacity=".25"><path d="M53.783 104.537c-.239 0-.48-.072-.691-.219a1.21 1.21 0 01-.29-1.684c.273-.387.519-.799.73-1.227a9.028 9.028 0 00-.501-8.961 1.208 1.208 0 112.032-1.307c2.176 3.389 2.42 7.736.633 11.34a11.833 11.833 0 01-.92 1.547 1.204 1.204 0 01-.993.511M59.339 107.414a1.207 1.207 0 01-.968-1.922c.453-.615.854-1.279 1.198-1.973a13.243 13.243 0 00-1.251-13.846 1.207 1.207 0 111.937-1.441 15.643 15.643 0 011.478 16.357c-.404.818-.88 1.604-1.414 2.33a1.213 1.213 0 01-.98.495M48.345 101.486a1.21 1.21 0 01-1.045-1.802 4.802 4.802 0 00.18-4.369 1.208 1.208 0 012.193-1.01 7.22 7.22 0 01-.271 6.57 1.209 1.209 0 01-1.057.611"/></g><g fill="#EB743B"><path d="M53.783 101.537c-.239 0-.48-.072-.691-.219a1.21 1.21 0 01-.29-1.684c.273-.387.519-.799.73-1.227a9.028 9.028 0 00-.501-8.961 1.208 1.208 0 112.032-1.307c2.176 3.389 2.42 7.736.633 11.34a11.833 11.833 0 01-.92 1.547 1.204 1.204 0 01-.993.511M59.339 104.414a1.207 1.207 0 01-.968-1.922c.453-.615.854-1.279 1.198-1.973a13.243 13.243 0 00-1.251-13.846 1.207 1.207 0 111.937-1.441 15.643 15.643 0 011.478 16.357c-.404.818-.88 1.604-1.414 2.33a1.213 1.213 0 01-.98.495M48.345 98.486a1.21 1.21 0 01-1.045-1.802 4.802 4.802 0 00.18-4.369 1.208 1.208 0 012.193-1.01 7.22 7.22 0 01-.271 6.57 1.209 1.209 0 01-1.057.611"/></g><g><path d="M66.082 47.24c-10.12 0-20.365 3.656-26.736 9.543-6.886 6.361-10.521 16.245-10.348 27.989a3.99 3.99 0 00-3.41 3.941v18.119a4 4 0 004 4h2.033c.316 0 .619-.047.914-.115v1.602c0 2.209 1.565 4 3.495 4h1.898c1.93 0 3.495-1.791 3.495-4V82.65c0-2.209-1.565-4-3.495-4H36.03c-1.717 0-3.138 1.42-3.432 3.288.065-1.855.204-3.652.435-5.096 1.154-7.232 4.119-13.268 8.688-17.488 5.664-5.232 15.226-8.613 24.361-8.613 1.042 0 3.241.043 4.277.143v-3.49c-1.2-.117-3.163-.154-4.277-.154zm-34.459 59.188a2.652 2.652 0 01-2.651-2.65v-12.01a2.651 2.651 0 012.651-2.652h.913v17.312h-.913z" opacity=".25"/><path fill="#FFFEDB" d="M66.082 44.24c-10.12 0-20.365 3.656-26.736 9.543-6.886 6.361-10.521 16.245-10.348 27.989a3.99 3.99 0 00-3.41 3.941v18.119a4 4 0 004 4h2.033c.316 0 .619-.047.914-.115v1.602c0 2.209 1.565 4 3.495 4h1.898c1.93 0 3.495-1.791 3.495-4V79.65c0-2.209-1.565-4-3.495-4H36.03c-1.717 0-3.138 1.42-3.432 3.287.065-1.854.204-3.651.435-5.096 1.154-7.232 4.119-13.268 8.688-17.488 5.664-5.232 15.226-8.613 24.361-8.613 1.042 0 3.241.043 4.277.143v-3.49c-1.2-.116-3.163-.153-4.277-.153zm-34.459 59.188a2.652 2.652 0 01-2.651-2.65v-12.01a2.651 2.651 0 012.651-2.652h.913v17.312h-.913z"/></g></svg>
```

**Review by MonkeyDo — CHANGES_REQUESTED** [review]
A small tweak, apart from that it's good to go !

**Review by MonkeyDo — APPROVED** [review]
🚀

**Comment by mayhem** [comment]
Thank you!

**Comment by mayhem** [comment]
I'll push this live first week in Oct when I update the metabrainz page as per schedule.

**Pull request merged**